libconnman-qt/networkservice.cpp

```diff
@@ -475,7 +475,9 @@ void NetworkService::reconnectServiceInterface()
     connect(m_service, SIGNAL(PropertyChanged(QString,QDBusVariant)),
             this, SLOT(updateProperty(QString,QDBusVariant)));
 
-    if (m_path != QStringLiteral("/") && m_service->isValid()) {
+    if (state().isEmpty()) //saved services have an empty state and cached properties
+        QTimer::singleShot(500,this,SIGNAL(propertiesReady()));
```

Comment:
Arbitrary. This is what it was before it got properties instead of just using cached properties from the manager.